core-framework/c-api-framework/src/core/ProcessSession.cpp

```diff
@@ -100,7 +100,7 @@ void ProcessSession::read(FlowFile& flow_file, const io::InputStreamCallback& ca
   }
 }
 
-void ProcessSession::setAttribute(FlowFile& ff, std::string_view key, std::string value) {
+void ProcessSession::setAttribute(FlowFile& ff, std::string_view key, std::string value) {  // NOLINT(performance-unnecessary-value-param)
```

Comment:
I think we should just ignore this the signature comes from the parent class where it makes sense as you can move into it.